src/firmware/app/control/trajectory_planner_private.c

```diff
@@ -1,10 +1,9 @@
-#include "firmware/app/control/trajectory_planner_impl.h"
-
 #include <assert.h>
 #include <float.h>
 #include <math.h>
 
 #include "firmware/app/control/trajectory_planner.h"
+#include "firmware/app/control/trajectory_planner_impl.h"
```

Comment:
```suggestion
#include "firmware/app/control/trajectory_planner_private.h"
```
Build is failing cause it cant find the _impl files should change these to use the renamed files